src/forms/TopComponent/TopComponent.styled.ts

```diff
@@ -11,7 +11,7 @@ const RESPONSIVE_BREAKPOINT = BREAKPOINTS.topComponent.responsiveView + "px";
 
 export const FormContent = styled.div`
 	& .layout {
-		padding: ${pr => pr.view === "DRAWER" ? "15px 40px" : (pr.view === "RESPONSIVE" || pr.view === "DESKTOP" || pr.view === "BIG_DESKTOP") ? "15px 20px 0px 20px" : "0px"};
+		padding: ${pr => pr.view === "DRAWER" ? "15px 40px" : (pr.view === "RESPONSIVE" || pr.view === "DESKTOP" || pr.view === "BIG_DESKTOP") ? "24px" : "0px"};

```

Comment:
The idea here is that all variants of forms will have 24px around their layout. The only variant that might not is the mobile because of the reduced screen-size. 
```suggestion
		padding: ${pr => pr.view === "MOBILE" ? "0px" :  "24px"};

```